### ISSUE TITLE
Parameter Intellisense should include parameter aliases

### ISSUE THREAD

**Issue by JustinGrote** [PowerShell/vscode-powershell#3798]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.

### Summary

Example:
Get-Childitem -System has a parameter alias so it can also be referenced as -as, this should show up as a valid autocomplete, and maybe have an option to auto-translate the parameter to it's non-aliasesd name (though format-on-save should do this so manybe not necessary)

![image](https://user-images.githubusercontent.com/15258962/151671717-3e2e3e7b-2474-43e8-9406-05b6de502bce.png)
<The above should also show "as", maybe with an indicator that shows it's an alias to System>

**Comment by JustinGrote** [comment]
@SeeminglyScience fair point, maybe have it go ahead and retrieve aliases that start with that same initial character so that it can continue to filter? I can't remember if for a completion parameter you can have both a "display" and an "actual" value.

**Comment by SeeminglyScience** [comment]
Issue is we get all of our intellisense directly from PowerShell (`SMA.CommandCompletion.CompleteInput` method). So if we want to do something different than what PowerShell does already, we have to reimplement *a lot* of what it does before we can get to a point where we can inject something of our own.
